client/src/pages/about/index.js

```diff
@@ -74,30 +72,67 @@ export const About = () => {
         width={{ width: 'full', max: 'xlarge' }}
       >
         <Box pad={responsive({ horizontal: 'medium' })}>
-          <Text weight="bold">
-            ALSF funds cutting-edge pediatric cancer research
-          </Text>
-          <Text>
-            Alex’s Lemonade Stand Foundation (ALSF) funded 10 childhood cancer
-            investigators from eight different institutions working on
-            single-cell profiling to create a publicly available atlas of
-            single-cell pediatric cancer data.{' '}
-            <Link
-              href={config.links.grants_program}
-              label="Learn more about the grants
+          <Box margin={{ bottom: responsive('xlarge', '76px') }}>
+            <Text weight="bold">
+              ALSF funds cutting-edge pediatric cancer research
+            </Text>
+            <Text>
+              In 2019, Alex’s Lemonade Stand Foundation (ALSF) funded 10 grants
+              for investigators working on single-cell profiling of patient
+              samples to create a publicly available atlas of single-cell
+              pediatric cancer data. .
+            </Text>
+            <Text margin={{ top: 'medium' }}>
+              <Link
+                href={config.links.grants_program}
+                label="Learn more about the grants
             program"
-            />
-            .
-          </Text>
+              />
+            </Text>
+          </Box>
+          <Box margin={{ bottom: responsive('xlarge', '76px') }}>
+            <Text weight="bold">The Data Lab processes the generated data</Text>
+            <Text>
+              In 2019, Alex’s Lemonade Stand Foundation (ALSF) funded 10 grants
+              for investigators working on single-cell profiling of patient
+              samples to create a publicly available atlas of single-cell
+              pediatric cancer data.
+            </Text>
+            <Text margin={{ top: 'medium' }}>
+              <Link
+                href={config.links.scpca}
```

Comment:
That link should open in a new tab but I'm not qualified to speak on how it should be done. (i.e., rerouting)

Reply:
> That link should open in a new tab

Got it, thank you! 

@davidsmejia Do you want to use `/` and update our `Link` component? Pls let me know!

Reply:
Yeah, I think just adding a new prop to Link makes sense. 

The old convention was that new tabs were only open for any external link. We should preserve that.

What do you think about adding `newtab` prop that default to `false` and then in the check of the `Link` component check if we explicitly ask for a new tab? I like this solution because of the minimal change and it supports existing behavior. I think any other refactoring of the component should be in a new issue/pr.

Example:
```js
export const Link = ({
  ...
  newtab = false <---- new prop
}) => {
  return isExternalPath(href) || newtab ? <---- force new tab if explicitly asked
    <LinkAnchor />
  ) : (
    <NextLink href={href} legacyBehavior>
      <LinkAnchor />
    </NextLink>
  )
}

```

Reply:
Yes, that sounds good. I made this change and this PR is ready for another look (e8f2a2d4fc2b4942865ecfd6fd9746576b01975c)!